test/helpers/cmd.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"k8s.io/client-go/util/jsonpath"
+)
+
+//CmdRes Command response with all the data
+type CmdRes struct {
+	cmd    string
+	params []string
+	stdout *bytes.Buffer
+	stderr *bytes.Buffer
+	exit   bool
+}
+
+//WasSuccessful returns true if the command was sucessful
+func (res *CmdRes) WasSuccessful() bool {
+	return res.exit
+}
+
+//CountLines return the number of stdout lines
+func (res *CmdRes) CountLines() int {
+	return len(strings.Split(res.stdout.String(), "\n"))
+}
+
+//IntOutput returns the stdout of res as integer
+func (res *CmdRes) IntOutput() (int, error) {
+	return strconv.Atoi(strings.Trim(res.stdout.String(), "\n"))
+}
+
+//SingleOut returns the stdout of res without any newline characters
+func (res *CmdRes) SingleOut() string {
+	return strings.Trim(res.stdout.String(), "\n")
```

Comment:
I think this is supposed to be `return strings.Replace(res.stdout.String(), "\n", "", -1)`.